bbb-common-web/src/main/java/org/bigbluebutton/api/model/shared/Checksum.java

```diff
@@ -9,7 +9,7 @@
     @NotEmpty(message = "You must provide the API call")
     protected String apiCall;
 
-    @NotEmpty(message = "You must provide the checksum")
+    @NotEmpty(message = "checksumError-You must provide the checksum")

```

Comment:
Looks like this one was missed in the conversion to separate key & message fields.